scripts/README.md

```diff
@@ -0,0 +1,41 @@
+# Nuxeo Platform scripts
+
+* fixeclipse.py: Fix eclipse classpath & .ok files
+* gitf.bat/gitfunctions.sh: Convenient functions for use on Nuxeo projects version controlled under Git
+* nxutils.sh: Convenient functions for use on Nuxeo projects
+* release.py: Manages releasing of Nuxeo source code
+* release_mp.py: Manages releasing of Nuxeo Marketplace packages
+
+# Installation
+ - (Optional) Create a Python virtalenv
+ - Install requirements with pip install -r requirements.txt
+# Usage
+`./<script> <options>`
```

Comment:
* Call `fixeclipse` after an execution of `mvn eclipse:eclipse`
* Use `gitf` (or `gitfa`) to execute recursive Git actions on Nuxeo repositories (including addons)
* See `release.py -h` and `release_mp.py -h`